pkg/k8s/code/crud.go

```diff
@@ -0,0 +1,71 @@
+package code
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var devTerminationGracePeriodSeconds int64
+
+//Deploy creates or updates a database
+func Deploy(dev *model.Dev, c *kubernetes.Clientset) error {
+	ss := translate(dev)
+
+	if exists(ss, c) {
+		if err := update(ss, c); err != nil {
+			return err
+		}
+	} else {
+		if err := create(ss, c); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func exists(ss *appsv1.StatefulSet, c *kubernetes.Clientset) bool {
+	ss, err := c.AppsV1().StatefulSets(ss.Namespace).Get(ss.Name, metav1.GetOptions{})
+	if err != nil {
```

Comment:
I've seen a Get on deployments return a 'not-found' error instead of an empty name. do we need to check for that here as well?